src/launcher/main.cpp

```diff
@@ -508,10 +553,30 @@ int main(int argc, const char** argv) {
         signal(SIGINT, SIG_DFL);
         fprintf(stderr, "Done\n");
 
-        run_jattach(pid, String("stop,file=") << file << "," << output << format << ",log=" << logfile);
+        run_jattach(pid, kJattachLoad, String("stop,file=") << file << "," << output << format << ",log=" << logfile);
+    } else if (action == "fdtransfer") {
+        if (params != kEmpty) {
+            fprintf(stderr, "Warning: action fdtransfer was given, these parameters will be ignored: %s\n", params.str());
```

Comment:
Let's make it an error case (i.e `return 1` here). This is only for our usage (gprofiler) so no reason not to be very strict here.

Reply:
👍